### PR TITLE
[v4 upgrade] UX/UI manifest selection panel

### PR DESCRIPTION
**Pull request by ghost**
new UI panel for people to select V4/V3 default manifest and discovery files

image

### CODE REVIEW

**Comment by ghost** [comment]
missing swagger files for v4 which is reaquired for local tests runs

File: pkg/discovery/templates/ob-v4.0-ozone-headless.json [hunk 1]
```diff
@@ -0,0 +1,297 @@
...
+    "description": "O3 Headless consent flow.. An Ozone discovery template for v4.0 of Accounts and Payments with pre-populated model Bank (Ozone) data.",
+    "discoveryVersion": "v0.4.0",
+    "tokenAcquisition": "headless",
+    "discoveryItems": [
+      {
+        "apiSpecification": {
+          "name": "Account and Transaction API Specification",
+          "url": "https://openbanking.atlassian.net/wiki/spaces/DZ/pages/937820271/Account+and+Transaction+API+Specification+-+v3.1",
+          "version": "v3.1.0",
+          "schemaVersion": "https://raw.githubusercontent.com/OpenBankingUK/read-write-api-specs/v3.1.0/dist/account-info-swagger.json",
```
Comment (OBPeteS) [inline pkg/discovery/templates/ob-v4.0-ozone-headless.json]: schemaVersion points to 3.1.0

File: web/src/views/Wizard/WizardContinueOrStart.vue [hunk 2]
```diff
@@ -35,13 +35,32 @@
     <div class="d-flex align-items-end">
       <div class="panel w-100">
         <div class="panel-heading">
-          <h5>Or choose from a selection of discovery templates to begin.</h5>
+          <h5>V4.0 Discovery Templates</h5>
```
Comment (ghost) [inline web/src/views/Wizard/WizardContinueOrStart.vue:38]: we could look to toggle these panels based on the navbar select box. When people select Versions v4.* this div is shown and when they select any versions v3.* the V3 div is shown..

**Review by OBPeteS — APPROVED** [review]

File: pkg/discovery/templates/ob-v4.0-ozone-headless.json (hunk 1 above)
Comment (ghost) [reply pkg/discovery/templates/ob-v4.0-ozone-headless.json]: fixed

**Pull request merged**